Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/OnConfirm.hs

```diff
@@ -248,7 +88,7 @@ tfItems :: DConfirm.DConfirmResp -> Maybe [Spec.Item]
 tfItems res =
   Just
     [ Spec.Item
-        { itemDescriptor = tfItemDescriptor res,
+        { itemDescriptor = Nothing,
```

Comment:
Why is this removed ?

Reply:
It was not consumed at BAP & it's value is as same as `itemId`.

Reply:
This will be added in next change